src/v/serde/json/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//bazel:build.bzl", "redpanda_cc_library")
```

Comment:
>Also note that numbers are only written with 6 digits of precision. I don't really think we care about more than that.

Just curious about the context of this PR: the CL mentions the upcoming JSON support for Iceberg topics -- is that the main motivation here, or is there another upcoming use case?

Reply:
There are two use cases for this in my mind:

1) support well know values for struct and value in protobuf for iceberg by jsonifying it
2) usage in a v2 of the admin api that I have been slowly working on the last couple of months when I have a spare moment